### DISCUSSION TITLE
ツール/ドキュメント上での表記/翻訳揺れへの対処

### DISCUSSION THREAD

**Discussion by Sayamame-beans** [anatawa12/AvatarOptimizer#345] [General]
例えば日本語設定でのSkinnedMeshRendererに関するツール内表記では、日本語版Unityの表記である`スキンメッシュレンダラー`としている箇所と`SkinnedMeshRenderer`のままとしている箇所が混在しているように見えます。
~~多分私が修正するの忘れました。~~

そもそも日本語版Unityに揃えるべきなのかどうかという疑問もあります。
(英語以外の表記を有効にして利用したいユーザーは、Unityの英語表記に馴染みがある層ではない場合がそれなりに多そうというのを何となく思っています。恐らくその意図で、私は日本語版Unityの表記に寄せに行ったはずです。)

訳し方を安定させるために、これを行う際には結論を出しておきたい気持ちがあります。(意見・翻訳提案を緩く募集しています。)

This change is not planned for now.

**Comment by anatawa12** [comment]
とりあえず議論になりそうなのでconvert to discussionします

**Comment by Sayamame-beans** [comment]
ブレンドシェイプとBlendShapeでも表記が揺れていますね…

結論: `BlendShape` に統一

**Reply by Sayamame-beans** [reply]
BlendShapeに統一しました。

**Reply by Sayamame-beans** [reply]
なんと、"Blend Shape"という表記揺れもあるようです…
未対処です。

**Reply by Sayamame-beans** [reply]
もしかして、"Blend Shape"表記の方が正規なのでは?

**Reply by Sayamame-beans** [reply]
…AAOでは"BlendShape"に統一することにしようかなという気持ちになってきました
ので、しました

**Comment by Sayamame-beans** [comment]
https://misskey.niri.la/notes/9jbcivbn0v
https://misskey.niri.la/notes/9jba38sus0
から判断するに、日本語Unityにはそこまで寄せなくても良いのかなと感じています。
ドキュメント上では、各ページで一回は括弧書きで補足する、というのはありかもしれません(VRChatのTooltip翻訳のように)

**Reply by Sayamame-beans** [reply]
この観点からは、スキンメッシュレンダラー等はSkinned Mesh Rendererとかに戻さないとですね…

**Reply by Sayamame-beans** [reply]
> レファレンスマニュアル部分はある程度わかっててドキュメントが読める人(単語は調べる人)を想定したい。メンテコスト的にも。
> 
> トップページやbasic usageチュートリアルはアバタ改変初心者くらいまで下げたほうが良さそうって思ってます
https://github.com/anatawa12/AvatarOptimizer/issues/573#issuecomment-1760125234

**Comment by Sayamame-beans** [comment]
除去 または 削除
新しいチュートリアルのPRレビュー中に疑問を持ちました
"除去"である必要性…?

**Reply by Sayamame-beans** [reply]
blendshapeの"固定・除去"での"除去"は、BlendShapeが無くなるというイメージを明確に与えるのに一役買っている気がする
固定でも操作不可能になるイメージはあるので十分とも考えられますが、無い方が良いとまでは言えないのかな…という感じ

それ以外の箇所では"削除"が使われている場合が殆どである

**Reply by Sayamame-beans** [reply]
固定・除去 と 削除 の2パターンだけ残すようにしてみる。
Freeze Blendshape系には固定・除去(固定と除去)を、それ以外には削除を使えば良さそうなので…

**Comment by Sayamame-beans** [comment]
結合 vs 統合

結論: `統合` に統一

**Reply by Sayamame-beans** [reply]
これについては、"統合"に統一して良いと考えました。

**Comment by Sayamame-beans** [comment]
`!`の半角と全角が混ざっている模様
日本語ドキュメント(チュートリアル)は全角、ja.poは混ざっています。

結論: 日本語は全角、英語は半角で統一

**Reply by Sayamame-beans** [reply]
とりあえず日本語は全角で調整します。(元々そのつもりで認識していたのですけど、どこかで忘れてて混ざったんだと思います)

**Reply by Sayamame-beans** [reply]
統一

**Comment by Sayamame-beans** [comment]
[docs] コンポーネントの設定について、箇条書きのものと段落が分かれているものの両方がある

**Reply by Sayamame-beans** [reply]
例
Trace and OptimizeやMerge Boneは箇条書き
Merge Skinned Meshは段落が別

**Comment by Sayamame-beans** [comment]
add か attachか
イメージは後者だけど今よく使ってるのは前者かも

**Reply by anatawa12** [reply]
イメージattachだけど公式のボタンはAdd componentだからaddなのかなぁ

**Reply by Sayamame-beans** [reply]
"付ける"(動詞)としてはaddを使って、付いてるものや付いた状態を指す時はattach(attached)を使うとバランスが良さそう?

**Comment by Sayamame-beans** [comment]
些細ですが、"できる"と"出来る"が混ざっていました(殆ど前者にしてるっぽい)

**Comment by Sayamame-beans** [comment]
[docs]
hintを使ってる場所と"備考"で章分けしてる場所がある

**Comment by Sayamame-beans** [comment]
Trace and Optimizeの項目について、"自動的に"(Automatically)があったりなかったりしている
ref: https://github.com/anatawa12/AvatarOptimizer/pull/952/files#r1541037067

**Comment by Sayamame-beans** [comment]
"Avatar Optimizer" vs "AvatarOptimizer" in docs/description

**Reply by anatawa12** [reply]
AAO: Avatar Optimizerなのでスペースありかな

**Comment by Sayamame-beans** [comment]
Animator vs アニメーター

**Comment by Sayamame-beans** [comment]
アバタールート vs アバターのルート は"アバターのルート"に統一したようです
